src/main/java/org/delivery/KiwiEats/models/ProductDTO.java

```diff
@@ -0,0 +1,43 @@
+package org.delivery.KiwiEats.models;
+
+import jakarta.validation.constraints.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.delivery.KiwiEats.entities.Category;
+import org.hibernate.validator.constraints.Length;
+import org.hibernate.validator.constraints.URL;
+
+import java.sql.Timestamp;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ProductDTO {
+    @NotNull
```

Comment:
no need for this because it will always have the ID due to database providing a non null value no matter what